### PR TITLE
GraknEngineConfig handles unavailable properties

### PR DESCRIPTION
**Pull request by alexandraorth**

### CODE REVIEW

**Comment by alexandraorth** [comment]
please retest this please - weird jenkins thing

**Comment by alexandraorth** [comment]
please retest this please - `Command execution failed.`

File: grakn-engine/src/main/java/ai/grakn/engine/GraknEngineConfig.java [hunk 1]
```diff
@@ -237,20 +237,24 @@ public Properties getProperties() {
     }
 
     public String getProperty(String property) {
-        return prop.getProperty(property);
+         if(prop.containsKey(property)) {
```
Comment (fppt) [inline grakn-engine/src/main/java/ai/grakn/engine/GraknEngineConfig.java]: It bothers me so much that this is 3 lines and. . . . [*]

File: grakn-engine/src/main/java/ai/grakn/engine/GraknEngineConfig.java [hunk 2]
```diff
@@ -237,20 +237,24 @@ public Properties getProperties() {
...
     }
 
     public String getProperty(String property, String defaultValue) {
-        String res = prop.getProperty(property);
-        if(res != null) {
-            return res;
-        }
-
-        return defaultValue;
+        return prop.containsKey(property) ? prop.getProperty(property)
```
Comment (fppt) [inline grakn-engine/src/main/java/ai/grakn/engine/GraknEngineConfig.java]: [*] . . . this is 2 lines

File: grakn-engine/src/main/java/ai/grakn/engine/GraknEngineConfig.java (hunk 1 above)
Comment (alexandraorth) [inline grakn-engine/src/main/java/ai/grakn/engine/GraknEngineConfig.java]: This bothered me as well :( After the `:` side I tried to put the throw, but java doesn't let you do that.

**Pull request merged**